Sprint-2/implement/querystring.js

```diff
@@ -4,13 +4,18 @@ function parseQueryString(queryString) {
     return queryParams;
   }
   const keyValuePairs = queryString.split("&");
-
+  console.log(keyValuePairs);
   for (const pair of keyValuePairs) {
-    const [key, value] = pair.split("=");
+    const index1 = pair.indexOf("=");
+    console.log(index1);
+    const key = pair.slice(0, index1);
+    const value = pair.slice(index1 + 1);
+    console.log(value);
+    console.log(key);
```

Comment:
For each of the following function calls, does your function return the value you expected?
```
parseQueryString("a=b&=&c=d")
parseQueryString("=b")
parseQueryString("a=b&&c=d")
parseQueryString("a%25b=c%26d`)
```
Note: the `%25` and `%26` are ***URL encoded*** or ***percent encoded*** characters.